.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "mongo-ruby-driver"]
+	path = mongo-ruby-driver
+	url = https://github.com/mongodb/mongo-ruby-driver.git
+  branch = 2.18-stable
```

Comment:
the indentation went weird here, but it still seems to work…

Reply:
I think git seems to have two different behaviors for indenting when using "git submodule add -b" vs "git checkout" in the submodule directory. I fixed the indentation so that it matches for all lines.